### PR TITLE
Adjust script to recent class changes

### PR DESCRIPTION
**Pull request by marcelveldt**

### CODE REVIEW

File: zwave_js_server/__main__.py [hunk 1]
```diff
@@ -62,9 +62,9 @@ def log_value_updated(event: dict) -> None:
         value = event["value"]
 
         if node.device_config:
-            description = node.device_config["description"]
+            description = node.device_config.description
         else:
-            description = f'{node.device_class["generic"]} (missing device config)'
+            description = f'{node.device_class.generic} (missing device config)'
```
Comment (MartinHjelmare) [inline zwave_js_server/__main__.py:67]: Should we guard against a None value here on `node.device_class.generic`?

File: zwave_js_server/__main__.py [hunk 2]
```diff
@@ -62,9 +62,9 @@ def log_value_updated(event: dict) -> None:
         value = event["value"]
 
         if node.device_config:
-            description = node.device_config["description"]
+            description = node.device_config.description
```
Comment (MartinHjelmare) [inline zwave_js_server/__main__.py:65]: This may return `None`. Maybe that's ok.

Comment (marcelveldt) [reply zwave_js_server/__main__.py:65]: yeah, no problem

File: zwave_js_server/__main__.py (hunk 1 above)
Comment (marcelveldt) [reply zwave_js_server/__main__.py:67]: should always be present

Comment (MartinHjelmare) [reply zwave_js_server/__main__.py]: It's typed as Optional in the device class.

Comment (MartinHjelmare) [reply zwave_js_server/__main__.py]: I'll fix this tomorrow. Merging as is.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**